clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/callbacks/Utils.groovy

```diff
@@ -47,6 +47,16 @@ class Utils {
     return lastIndex != -1 ? fullUrl.substring(lastIndex + 1) : fullUrl
   }
 
+  static String getTargetProxyType(String fullUrl) {
+    if (!fullUrl) {
+      return fullUrl
+    }
+
+    int lastIndex = fullUrl.lastIndexOf('/')
+    String withoutName = lastIndex != -1 ? fullUrl.substring(0, lastIndex) : fullUrl
```

Comment:
If the URL doesn't have a `'/'`, how do you know that means it doesn't include the name?